docs/tests/plugins/datalogger/datalogger_test.rst

```diff
@@ -0,0 +1,1190 @@
+.. _datalogger_tests:
+
+Datalogger Test Suite
+================================================================================
+
+.. note::
+
+  User guide :ref:`Data Logger user guide <datalogger>`
+
+The following apply to all tests below, any special requirements will be mentioned.
+
+**Prerequisites:**
+    * Scopy v2.0.0 or later with Data Logger installed on the system
+    * Tests listed as dependencies are successfully completed
+    * Reset .ini files to default by pressing "Reset" button in Preferences
+
+Setup
+-------
+
+.. _adalm_pluto_emu_setup:
```

Comment:
These need to be unique. So we either change them in this PR or in this one: https://github.com/analogdevicesinc/scopy/pull/1868/files#diff-765eec5eb0d2ba178cc39341f3139e4a27148112e6de2cbc3ea3aa72bfa4661fR18-R27 .

Otherwise, when we merge everything together, the docs job will fail due to duplicate tags.